flink-core/src/main/java/org/apache/flink/api/java/typeutils/runtime/PojoSerializer.java

```diff
@@ -193,7 +193,7 @@ public T createInstance() {
 			return null;
 		}
 		try {
-			T t = clazz.newInstance();
+			T t = clazz.getDeclaredConstructor().newInstance();
```

Comment:
this change may have performance implications as noted in the mail thread linked in the jira.